api/v1/models/Team.js

```diff
@@ -0,0 +1,48 @@
+/* jshint esversion: 6 */
+var _ = require('lodash');
+
+var database = require('../../database');
+var bookshelf = database.instance();
+
+var Model = require('./Model');
+var Team = Model.extend({
+	tableName: 'teams',
+	idAttribute: 'id',
+	hasTimestamps: ['created', 'updated'],
+	validations: {
+		name: ['required', 'string', 'maxLength:255'],
+		size: [	'required',
+						'integer',
```

Comment:
Not intentional. I don't know why it appears like that. Looks normal in Atom. I'll make sure to run Atom's auto-indent on all the files when I make fixes.